Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+            args '-u root:root --cap-add SYS_PTRACE -v "/tmp/gomod":/go/pkg/mod'
+        }
+    }
+    options {
+          timeout(time: 26, unit: 'MINUTES')
+    }
+    environment {
+        NPM_CONFIG_CACHE = '/tmp/npm/cache'
+    }
+    stages {
+        stage('Build') {
+            steps {
+                checkout scm
+                sh 'mkdir -p .build'
+            }
+        }
+
+        stage('Verification') {
+            parallel {
+                stage('Lint') {
+                    steps {
+                        sh 'check-copyright'
+                        sh 'check-large-files'
+                        sh 'check-imports ./...'
+                        sh 'check-peer-constraints'
+                        sh 'storj-protobuf --protoc=$HOME/protoc/bin/protoc lint'
+                        sh 'storj-protobuf --protoc=$HOME/protoc/bin/protoc check-lock'
+                        sh 'check-atomic-align ./...'
+                        sh 'check-errs ./...'
+                        sh 'staticcheck ./' // we don't want to check the generated code
+                        sh 'golangci-lint --config /go/ci/.golangci.yml -j=2 run'
+                        sh 'go-licenses check ./...'
+                    }
+                }
+
+// TODO: enable tests when we have tests
+/*                 stage('Tests') {
+                    environment {
+                        COVERFLAGS = "${ env.BRANCH_NAME != 'master' ? '' : '-coverprofile=.build/coverprofile -coverpkg=./...'}"
+                    }
+                    steps {
+                        sh 'go vet ./...'
+                        sh 'go test -parallel 4 -p 6 -vet=off $COVERFLAGS -timeout 20m -json -race ./... 2>&1 | tee .build/tests.json | xunit -out .build/tests.xml'
+                        // TODO enable this later 
+                        // sh 'check-clean-directory'
+                    }
+
+                    post {
+                        always {
+                            sh script: 'cat .build/tests.json | tparse -all -top -slow 100', returnStatus: true
+                            archiveArtifacts artifacts: '.build/tests.json'
+                            junit '.build/tests.xml'
+
+                            script {
+                                if(fileExists(".build/coverprofile")){
+                                    sh script: 'filter-cover-profile < .build/coverprofile > .build/clean.coverprofile', returnStatus: true
+                                    sh script: 'gocov convert .build/clean.coverprofile > .build/cover.json', returnStatus: true
+                                    sh script: 'gocov-xml  < .build/cover.json > .build/cobertura.xml', returnStatus: true
+                                    cobertura coberturaReportFile: '.build/cobertura.xml'
+                                }
+                            }
+                        }
+                    } */
+                // }
```

Comment:
Why not just enable it now? And it'll run fast?

Reply:
I thought it would complain about code coverage. I just uncommented it

Reply:
Maybe try writing a single dummy test, so that it doesn't die on the test output?

Reply:
I think it's better to just be running it, so that we don't have to turn it on in the future.